### PR TITLE
Fix issue on task creation when is_private feature is disabled.

### LINKED ISSUE #5568: Adding tasks, when private_task feature disabled not possible
_Harvest: `[3.2.2.1] OneGov GEVER`: `Feature: Persönliche Aufgaben - OGIP 46`_
It displays an validation error, but the field is disabled, so no user input possible.

Bildschirmfoto 2019-04-17 um 10 20 22

### PR DESCRIPTION
**Pull request by phgross**
Omit the is_private field instead of only set it to hidden mode, when the feature is disabled. Fixes #5568

### CODE REVIEW

File: opengever/task/tests/test_private_task.py [hunk 1]
```diff
@@ -56,8 +57,21 @@ def test_is_private_field_is_hidden_in_add_form(self, browser):
         factoriesmenu.add('Task')
 
         self.assertEqual(
-            'hidden',
-            browser.css('#formfield-form-widgets-is_private input').first.type)
+            [],
+            browser.css('#formfield-form-widgets-is_private'))
+
+    @browsing
+    def test_adding_a_taks_with_disabled_feature_works(self, browser):
```
Comment (njohner) [inline opengever/task/tests/test_private_task.py]: typo `task` not `taks`

File: opengever/task/tests/test_private_task.py [hunk 2]
```diff
@@ -66,27 +80,8 @@ def test_is_private_field_is_hidden_in_edit_form(self, browser):
         browser.visit(self.task, view="edit")
 
         self.assertEqual(
-            'hidden',
-            browser.css('#formfield-form-widgets-is_private input').first.type)
-
-    @browsing
-    def test_cant_create_private_tasks_when_feature_is_inactive(self, browser):
-        self.login(self.dossier_responsible, browser)
```
Comment (njohner) [inline opengever/task/tests/test_private_task.py]: So is the invariant still tested anywhere? Maybe using the API?

Comment (phgross) [reply opengever/task/tests/test_private_task.py]: Yes it's still tested in the API tests. 

Passing in attributes for omitted fields, as its done it the removed tests, is ignored by z3c.form itself, so no need for this test.

**Comment by phgross** [comment]
@njohner updated.  Omitting the `revoke_permissions` field instead of only hiding it. For local consistency between the `is_private` setting and the `revoke_permissions` option.

File: opengever/task/tests/test_revoke_permissions.py [hunk 3]
```diff
@@ -276,27 +276,6 @@ def test_revoke_permissions_only_shown_when_feature_is_enabled_in_edit_form(self
...
-            form = browser.find_form_by_field('Responsible')
-            form.find_widget('Responsible').fill(
-                'fa:{}'.format(self.secretariat_user.getId()))
-
-            browser.css('#form-buttons-save').first.click()
-
-        self.assertEqual(['There were some errors.'], error_messages())
-        self.assertEqual(['The revoke permissions feature is disabled'],
-                         browser.css('form .error').text)
-        self.assertEqual(0, len(children['added']))
```
Comment (njohner) [inline opengever/task/tests/test_revoke_permissions.py]: Ok same thing as for the private_task, this is still tested in the API tests, so all good! 👍

**Pull request merged**